spec/rubocop/cop/rspec/named_subject_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::NamedSubject do
+  subject(:cop) { described_class.new }
+
+  it 'checks it for explicit subject usage' do
+    expect_violation(<<-SRC)
```

Comment:
Yeah good point. Using `<<-RUBY` also buys you syntax highlighting in many places